scripts/create-influx_debug-bucket.sh

```diff
@@ -3,3 +3,21 @@ set -e
 
```

Comment:
In general, it might be nice to have these bucket names in the environment variable file, and then this bash script would loop through them to create the extra buckets.

Reply:
Good Idea! One problem I am facing, however, is finding a way to modify the docker-compose.yaml to reflect these changes. Although currently, it seems that if both the script is not running and that the docker-compose.yaml environment variables are not being read correctly then the parser and data flow to influx and grafana **still** work just fine. We can keep this as a **TODO** to check in on later. Thoughts?

Example in docker-compose.yaml:

![image](https://github.com/UBC-Solar/sunlink/assets/117491745/639fe2e3-ce6d-4ef1-9b75-04dde1a4d339)

When I changed the `INFLUX_DEBUG_BUCKET` variable to `_test` (not an existing bucket) from `Test` (an existing bucket) in the environment file and then restarted the containers and ran the randomizer, it seemed that no problems occurred. Also, Influx did not create a bucket called `_test` either.